### PR TITLE
Navbar changed

### PR DESCRIPTION
**Pull request by Divyansh-Singhal**
Navbar changed

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -3,13 +3,18 @@ import "./Navbar.css"
...
-        <nav>
-            <a href="/">VIT Links! 💥</a>
-            <div className="nav-links">
-                <a href="/">Home</a>
-                <a href="/about">About</a>
-                <a href="/contact">Contact</a>
-            </div>
-        </nav>
+       <nav class="navbar navbar-expand-lg navbar-light bg-transparent">
+  <a class="navbar-brand" href="#"><img id="imga" src="/image-removebg-preview (1).png" ></img></a>
```
Comment (guptasajal411) [inline src/components/Navbar.jsx]: Can you change the `href="#"` to `href="/"`

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -3,13 +3,18 @@ import "./Navbar.css"
...
+  
+  <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+    <div id="navflr" class="navbar-nav">
+           <a class="nav-item nav-link" href="#">Home</a>
```
Comment (guptasajal411) [inline src/components/Navbar.jsx]: Can you change the `href="#"` to `href="/"` here too

File: src/components/Navbar.jsx (hunk 1 above)
Comment (Divyansh-Singhal) [reply src/components/Navbar.jsx]: Okay wait doing it

Comment (Divyansh-Singhal) [reply src/components/Navbar.jsx]: Changed Please review it.

**Pull request merged**